modules/gltf/README.md

```diff
@@ -2,4 +2,4 @@
 
 loaders.gl is a collection of framework independent 3D and geospatial loaders (parsers).
 
-This module contains loader for the GLTF format.
\ No newline at end of file
+This module contains loader and writers for the GLTF format.
```

Comment:
... , including GLB binary format decoder/encoders.

Nit: capitalization is `glTF`.